jack-and-the-beans/buddies-ios
# Filter for Date range and Location range from search bar

**Note:** The following is not included, this is a separate story. (The filter sliders will persist only until you leave the view)

> The "distance" and "date" filters in the search component should be persistent across all instances of the search component

To Test:
1. Navigate to Topics Tab
2. Select any Topic
3. Tap the "Filter" button on search bar
4. Drag sliders, and observe changes in search results after tapping "Filter"
